frontend/app/src/App.tsx

```diff
@@ -575,6 +576,12 @@ export class App extends PureComponent<Props, State> {
         scriptRunState: this.state.scriptRunState,
       })
     }
+    // Rerun script if the theme changed
+    if (
+      _prevProps.theme.activeTheme.name !== this.props.theme.activeTheme.name
+    ) {
+      this.sendRerunBackMsg()
+    }
```

Comment:
Does this also cover scenarios like the theme getting changed via host communication or via theme editor? And do we want to do something when the system theme changes during runtime? 

Also, is there a risk of redundant reruns -> e.g. does this lead to always having another rerun when an themed app sessions starts?